community/README.md

```diff
@@ -0,0 +1,62 @@
+# Nebula Graph Community Architecture
+
+The Nebula Graph community is organized as shown below:
+
+![Nebula Community Architecture](./images/structure.png)
+
+## PMC
+
+PMC (Project Management Committee) is the entity the controls and leads the whole nebula project. PMC members have the right to propose a new Maintainers or Committers, and the right to vote for the major decisions for Nebula Graph. See [PMC List](./pmc-list.md) for a list of PMC members.
+
+
+## Maintainer
+
+The planner and designer of the repository, with the right to merge branches into the master. A Maintainer should:
+
+- Set technical directions, roadmap, and priorities for the repository
+- Ensure the overall quality of the repository
+
+### How to become a Maintainer of a Repository
+
+- Must be a committer of the repository
+- Nominated by and obtained consensus approval from the PMC
+- The appointment is for one year
+
+See [Maintainer List](./maintainer-list.md) for the Maintainers of each repository.
+
+## Committer
+
+Recommended by Maintainer or Project Management Committee (PMC) as a Committer who has made a significant contribution to the repository. A committer has approval permission for code reviews of the repository. See [Committer List](./committer-list.md) for the Committers of each repository.
+
+> Note: Each repository requires at least 2 approvals for each PR to be merged into the master branch.
+
+### How to become a Committer of a Repository
+
+- Generated from Active Contributors.
+- Has more than 5 PRs merged to the master branch of the repository within a year.
+- Nominated by the Matainter of the repository or PMC and gained majority(1/2) votes from the Decision-Making Group(consists of Maintainers of the repository and the PMC).
+- The appointment is for one year.
+
+
+## Active Contributor
+
+Active Contributors are continuously active contributors in the community. They can have issues and PRs assigned to them, participate in the development. See [Active Contributor List](active-contributor-list.md) for the list of Active Contributors.
+
+### How to become an Active Contributor of a Repository
+
+- Contributes at least 5 PRs to a specific repository within one year will become an active contributor automatically.
```

Comment:
If you contribute at least 5 PRs to a specific repository within one year, you will become an ....